tests/test_hydra.py

```diff
@@ -617,6 +631,20 @@ def test_sweep_complex_defaults(
             ),
             id="overriding_help_template:$CONFIG,interp,yes_resolve",
         ),
+        param(
+            "examples/tutorials/basic/your_first_hydra_app/2_config_file/my_app.py",
+            ["--help", "--resolve"],
+            ["hydra.help.template=$CONFIG", "db.user=${hydra:job.name}"],
+            dedent(
+                """\
+                db:
+                  driver: mysql
+                  user: my_app
+                  password: secret
+                """
+            ),
+            id="overriding_help_template:$CONFIG,resolve_interp_to_hydra_config",
```

Comment:
generally speaking, avoid testing with the tutorial apps except in tests/test_examples. (I know the code is not being consistent about it).

Reply:
Made the change in https://github.com/facebookresearch/hydra/pull/1696/commits/27052cf96c4261051b4eea432e34ccbabbb1e22e.